node/tests/TestUtilities.ts

```diff
@@ -536,5 +537,21 @@ export async function transactionTest(
     args.push(1);
     baseTransaction.pfcount([key11]);
     args.push(3);
+    baseTransaction.rpush(key15, [
```

Comment:
nit: not required, but do we want to group these with the other list commands in this file?